### PR TITLE
chore: add security policy

### PR DESCRIPTION
**Pull request by adamledwards**
Potential security policy for the MOD org this was taken from [uktrade/.github ](https://github.com/uktrade/.github/blob/main/SECURITY.md)

### CODE REVIEW

File: SECRUITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+The Defence Digital Digital Standards Policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability, please submit your report to us using [here](#)
```
Comment (adamledwards) [inline SECRUITY.md:7]: We have two options here do we open a HackerOne account for the org or do we sent to a email address

File: SECRUITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- The website, IP or page where the vulnerability can be observed.
+- A brief description of the type of vulnerability, for example; "XSS vulnerability".
+- Steps to reproduce. These should be a benign, non-destructive, proof of concept. This helps to ensure that the report can be triaged quickly and accurately. It also reduces the likelihood of duplicate reports, or malicious exploitation of some vulnerabilities, such as sub-domain takeovers.
+
+Vulnerability reporting guidelines
+
+- Please do not share the vulnerability information beyond the owner and us, without express consent from the owner
+- Vulnerabilities reported to the HackerOne platform can be submitted without the need to create a HackerOne account. However, if you wish to be updated you should create an account
+- To submit your report, you will need to agree to the HackerOne Terms and Conditions and acknowledge that you have read their Privacy Policy and Disclosure Guidelines
+- Once you have submitted the report, it will be assessed by NCC Group within five working days, and forwarded to the affected owners as soon as possible.
```
Comment (adamledwards) [inline SECRUITY.md:19]: This is subject to change depending on which reporting method we choose to go with.

**Comment by adamledwards** [comment]
@chiefeh @jpveooys

At the moment we don't have a HackerOne account, is this the route we would like to take.?

File: SECRUITY.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+The Defence Digital Digital Standards Policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.
```
Comment (jpveooys) [inline SECRUITY.md:3]: IMO something like would do here:

```suggestion
This is the security notice for all Defence Digital repositories. The notice explains how vulnerabilities should be reported to us.
```

**Comment by jpveooys** [comment]
It looks like there's an existing policy here (with HackerOne): https://www.gov.uk/guidance/report-a-vulnerability-on-an-mod-system

So could just link to that perhaps?

**Comment by adamledwards** [comment]
I think using the link and MOJ method with just a link is the best way to go.

**Review by chiefeh — APPROVED** [review]

**Pull request merged**